docker/build.sh

```diff
@@ -164,25 +164,29 @@ if [[ $? != "0" ]]; then
     exit 1
 fi
 
-# Run the command inside the container.
-echo "Running '${COMMAND[@]}' inside ${DOCKER_IMG_SPEC}..."
-
-# By default we cleanup - remove the container once it finish running (--rm)
-# and share the PID namespace (--pid=host) so the process inside does not have
-# pid 1 and SIGKILL is propagated to the process inside (jenkins can kill it).
-echo ${DOCKER_BINARY}
-${DOCKER_BINARY} run --rm --pid=host \
-    -v ${WORKSPACE}:/workspace \
-    -w /workspace \
-    -e "CI_BUILD_HOME=/workspace" \
-    -e "CI_BUILD_USER=$(id -u -n)" \
-    -e "CI_BUILD_UID=$(id -u)" \
-    -e "CI_BUILD_GROUP=$(id -g -n)" \
-    -e "CI_BUILD_GID=$(id -g)" \
-    -e "CI_PYTEST_ADD_OPTIONS=$CI_PYTEST_ADD_OPTIONS" \
-    -e "CI_IMAGE_NAME=${DOCKER_IMAGE_NAME}" \
-    ${CUDA_ENV}\
-    ${CI_DOCKER_EXTRA_PARAMS[@]} \
-    ${DOCKER_IMG_SPEC} \
-    bash --login docker/with_the_same_user \
-    ${COMMAND[@]}
+if [[ -n ${COMMAND} ]]; then
+
+    # Run the command inside the container.
+    echo "Running '${COMMAND[@]}' inside ${DOCKER_IMG_SPEC}..."
+
+    # By default we cleanup - remove the container once it finish running (--rm)
+    # and share the PID namespace (--pid=host) so the process inside does not have
+    # pid 1 and SIGKILL is propagated to the process inside (jenkins can kill it).
+    echo ${DOCKER_BINARY}
```

Comment:
delete

Reply:
Hi @areusch , i don't get what i should delete. The enclosing ``if`` ... ``fi`` or just the ``echo ${DOCKER_BINARY}``?

Reply:
the comment snippet ``#By default ... echo ${DOCKER_BINARY}`` was already there from before. i this change i only wrapped it with the ``if ... fi``.

Reply:
oh just the echo, I think it's debug, right?

Reply:
oh, actually, this is fine to leave as-is, then.